pom.xml

```diff
@@ -214,12 +214,12 @@
 
       <plugin>
         <artifactId>maven-checkstyle-plugin</artifactId>
-        <version>2.17</version>
+        <version>3.0.1</version>
```

Comment:
I think it's 3.1.0, not 3.0.1: https://search.maven.org/search?q=maven-checkstyle-plugin

Reply:
done